pkg/autoscaler/multiscaler_test.go

```diff
@@ -80,12 +84,24 @@ func verifyNoTick(errCh chan error) error {
 	}
 }
 
+// verifyTick verifies that we get a tick in a certain amount of time.
+func verifyStatMessageTick(statsCh chan *StatMessage) error {
+	select {
+	case <-statsCh:
+		// We got the StatMessage!
+		return nil
+	case <-time.After(time.Second / samplesPerSecond):
```

Comment:
This looks flake inducing, should the timeout be twice the tick interval to be safe here?

Reply:
Done.